packages/theme-chalk/src/timeline.scss

```diff
@@ -16,4 +16,31 @@
       display: none;
     }
   }
+
+  & .#{$namespace}-timeline-item__center {
+    display: flex;
+    align-items: center;
+
+    & .#{$namespace}-timeline-item__wrapper {
```

Comment:
I think the `&` symbol is redundant here since in this scope, the `&` represents `#{$namespace}-timeline-item__center`, and in `scss` when you nest rules inside another one they become parent-child relationship

Reply:
Same goes the others, I think you might need to remove these `&` selector.

Reply:
Thanks for your help, I have fixed it